use-cases/using-SPDK-vhostuser-and-kata.md

```diff
@@ -0,0 +1,153 @@
+# Setup to run SPDK vhost-user devices with Kata Containers and Docker*
+
+- [SPDK vhost-user target overview](#spdk-vhost-user-target-overview)
+- [Install and setup SPDK vhost-user target](#install-and-setup-spdk-vhost-user-target)
+  - [Get source code and build SPDK](#get-source-code-and-build-spdk)
+  - [Run SPDK vhost-user target](#run-spdk-vhost-user-target)
+- [Host setup for vhost-user devices](#host-setup-for-vhost-user-devices)
+- [Launch a Kata container with SPDK vhost-user block device](#launch-a-kata-container-with-spdk-vhost-user-block-device)
+
+## SPDK vhost-user Target Overview
+
+The Storage Performance Development Kit (SPDK) provides a set of tools and
+libraries for writing high performance, scalable, user-mode storage applications.
+
+While, virtio is a efficient way to transport data for virtual environments and
+guests. It is most commonly used in QEMU VMs, where the QEMU itself exposes a
+virtual PCI device and the guest OS communicates with it using a specific Virtio
+PCI driver.
+vhost is a protocol for devices accessible via inter-process communication. It
+uses the same virtio queue layout as Virtio to allow vhost devices to be mapped
+directly to virtio devices. The initial vhost implementation is a part of the
+Linux kernel and uses ioctl interface to communicate with userspace applications.
+vhost-user implements the control plane through Unix domain socket to establish
+virtio queue sharing with a user space process on the same host. So SPDK can expose
+vhost device via vhost-user protocol.
+
+SPDK vhost is a vhost-user slave server. It exposes Unix domain sockets and
+allows external applications to connect. It is capable of exposing virtualized
+storage devices to QEMU instances or other arbitrary processes.
+
+Currently, SPDK vhost-user target can exposes these types of virtualized devices:
+
+- `vhost-user-blk`
+- `vhost-user-scsi`
+- `vhost-user-nvme`
+
+For more information, visit [SPDK](https://spdk.io) and [SPDK vhost-user target](https://spdk.io/doc/vhost.html).
+
+## Install and setup SPDK vhost-user target
+
+### Get source code and build SPDK
+
+Following SPDK [getting started](https://spdk.io/doc/getting_started.html)
+
+### Run SPDK vhost-user target
+
+First, run the SPDK setup.sh script to setup some hugepages for the SPDK vhost
```

Comment:
- `setup.sh`
- I'm assuming this script is part of some download in the "getting started" guide above? Are those instructions likely to change, or could we add a call to `curl -LO ...` to download the latest release in this doc to allow it to be tested (see below)?

Reply:
If download one SPDK release, then essential submodule, like DPDK, is also required to be downloaded. Also some dependent modules are required to install. So I'm afraid that it won't be briefer than getting-started doc.